GitHub.Collectors.Functions/GitHubFunctions.cs

```diff
@@ -706,6 +710,116 @@ public async Task ExecutePointCollectorAsync(string queueItem, ExecutionContext
             }
         }
 
+        [FunctionName("DiscoverOrganizations")]
+        public Task AutoOnboard([TimerTrigger("0 0 0 * * *") /* execute once per day at midnight*/] TimerInfo timerInfo, ExecutionContext executionContext, ILogger logger)
```

Comment:
>midnight [](http://example.com/codeflow?start=88&length=8)

Can we please check whether this is midnight UTC? If that is the case this becomes 4-5 PM PST. Nothing special just to note.

Reply:
Yes this would have executed at midnight UTC, I have modified for PST